### PR TITLE
Chronicles of Eberron

### PR DESCRIPTION
**Pull request by BucketListMod**
- Added Chronicles of Eberron (mechanics only)
- Updated/fixed Exploring Eberron

### CODE REVIEW

File: collection/Keith Baker; Chronicles of Eberron.json [hunk 1]
```diff
@@ -0,0 +1,5911 @@
...
+								]
+							]
+						}
+					]
+				}
+			]
+		}
+	],
+	"book": [],
+	"bookData": [],
```
Comment (TheGiddyLimit) [inline collection/Keith Baker; Chronicles of Eberron.json:806]:
```suggestion
```

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
looks like this could use some local testing + fixing

**Comment by BucketListMod** [comment]
> looks like this could use some local testing + fixing

I did local testing and fixing before committing the changes, worked fine on my device? Skimming through I found a few more small errors in ExE that I could fix and I could change the "psiactive" property from "special" to its own kind, but the pages function normally and all the mechanics work fine?

**Comment by TheGiddyLimit** [comment]
apologies, I mean specifically running `npm run test:json` locally; it's failing some schema checks

**Comment by TheGiddyLimit** [comment]
looks like you've now got some merge conflicts 🙈 you'll need to do a `git pull`

**Comment by BucketListMod** [comment]
Would it be smarter/easier to just close this one and open a new one? 🤣

**Pull request closed without merging**